Config.h

```diff
@@ -261,32 +286,58 @@
 					const int pin_led_tx = 25;
 				#endif
 			#endif
-		#else
-			#error An unsupported board was selected. Cannot compile RNode firmware.
-		#endif
-
-        bool mw_radio_online = false;
-
-		#define CONFIG_UART_BUFFER_SIZE 6144
-		#define CONFIG_QUEUE_SIZE 6144
-		#define CONFIG_QUEUE_MAX_LENGTH 200
-
-		#define EEPROM_SIZE 1024
-		#define EEPROM_OFFSET EEPROM_SIZE-EEPROM_RESERVED
-
-		#define GPS_BAUD_RATE 9600
-		#define PIN_GPS_TX 12
-		#define PIN_GPS_RX 34
+        #endif
+    #elif PLATFORM == PLATFORM_NRF52
+        #if BOARD_MODEL == BOARD_RAK4630
+            #define HAS_EEPROM false
+            #define HAS_DISPLAY false // set for debugging
+            #define HAS_BLUETOOTH false
+            #define HAS_CONSOLE false
+            #define HAS_PMU false
+            #define HAS_NP false
+            #define HAS_SD false
+            #define HAS_TCXO true
+            #define MODEM SX1262 
+            
+            #define CONFIG_UART_BUFFER_SIZE 6144
+            #define CONFIG_QUEUE_SIZE 6144
+            #define CONFIG_QUEUE_MAX_LENGTH 200
+            #define EEPROM_SIZE 4096
+            #define EEPROM_OFFSET EEPROM_SIZE+0xED000-EEPROM_RESERVED
+
+            // following pins are for the sx1262
+            const int pin_rxen = 37;
+            const int pin_reset = 38;
+            const int pin_cs = 42;
+            const int pin_sclk = 43;
+            const int pin_mosi = 44;
+            const int pin_miso = 45;
+            const int pin_busy = 46;
+            const int pin_dio = 47;
+            const int pin_led_rx = LED_BLUE;
+            const int pin_led_tx = LED_GREEN;
+        #endif
+	#else
+		#error An unsupported board was selected. Cannot compile RNode firmware.
 	#endif
 
+    bool mw_radio_online = false;
+
 	#if BOARD_MODEL == BOARD_TBEAM
 		#define I2C_SDA 21
 		#define I2C_SCL 22
 		#define PMU_IRQ 35
 	#endif
 
+
 	#define eeprom_addr(a) (a+EEPROM_OFFSET)
 
+    #if MODEM == SX1276 || MODEM == SX1278
+        SPIClass spiModem(pin_miso, pin_sclk, pin_mosi);
```

Comment:
I think running software SPI on platforms that do support hardware SPI is suboptimal. It'd be great if we can just use the default SPI class in this case. Currently the firmware doesn't build for any of the previous boards because `pin_miso` et al are not defined ;) Can you just use the normal HW SPI class and reference it to the same name as used in LoRa.cpp?

Reply:
You are absolutely right Mark, I apologise for not realising this. Smooth brain move honestly, I'll revert this for the ESP32 platforms now.

Reply:
Partially resolved in 793b53dd4cc71ba7695a31d50e8fef46f39d322b